src/pages/about-us/_data.js

```diff
@@ -0,0 +1,67 @@
+import { localize } from 'components/localization'
+import Shield from 'images/svg/about-us/shield.svg'
+import Star from 'images/svg/about-us/star.svg'
+import People from 'images/svg/about-us/people.svg'
+import Hands from 'images/svg/about-us/hands.svg'
+
+export const our_values = [
+    {
+        title: localize('Integrity'),
+        text: localize(
+            'We serve our customers with fairness and transparency. We settle all contracts by the book and speak plainly and truthfully.',
```

Comment:
Let's try to create any new files like these in TS/TSX if possible. You can refer some basic conventions we're following [here](https://wikijs.deriv.cloud/en/Frontend/deriv-static/06-implementation-guide/ts-convention).

Reply:
/about-us page is converted to TS now